setup.py

```diff
@@ -1,19 +1,30 @@
-from setuptools import setup
+import os
 
+from setuptools import setup, find_packages
 
-def readme():
-    with open('README.rst') as f:
+
+exec(open('auth_backends/_version.py').read())
+
+
+def read(*paths):
+    """Build a file path from `paths` and return the contents."""
+    with open(os.path.join(*paths), 'r') as f:
         return f.read()
 
 
 setup(
     name='edx-auth-backends',
-    version='0.1',
+    version=__version__,
     description='Custom edX authentication backends and pipeline steps',
-    long_description=readme(),
+    long_description=(
+        read('README.rst') + '\n\n' +
+        read('HISTORY.rst') + '\n\n' +
+        read('AUTHORS')
+    ),
     classifiers=[
         'Development Status :: 4 - Beta',
         'License :: OSI Approved :: GNU Affero General Public License v3',
+        'Programming Language :: Python',
```

Comment:
Why is Programming Language specified twice?  I'm not familiar with this type of setup.